app/code/Magento/Eav/Model/ResourceModel/Entity/Attribute/Collection.php

```diff
@@ -426,7 +439,7 @@ protected function _addSetInfo()
     /**
      * Ad information about attribute sets to collection result data
      *
-     * @return \Magento\Framework\Model\ResourceModel\Db\Collection\AbstractCollection
+     * @return $this
```

Comment:
```suggestion
     * @return \Magento\Framework\Model\ResourceModel\Db\Collection\AbstractCollection
```

Reply:
Why?

Reply:
I believe because it's still returns value from parent class, where declared `\Magento\Framework\Model\ResourceModel\Db\Collection\AbstractCollection`. I don't mind for reverting this change